src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
-body {
+// #region head
+body,
+h1,
+h2,
+p {
   margin: 0;
+  font-family: "Roboto", "Open Sans", sans-serif;
+  color: #060b35;
 }
+// #endregion
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+// #region card head
+.product-card__image {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.product-card__product-code {
+  margin-top: 4px;
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+// #endregion
+// #region card reviews
+.reviews {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__stars {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+}
```

Comment:
Let's get rid from unnecessary blocks.
[BEM] - Don't make BEM structure too complicated. You don't need to create separate blocks for anything other than card itself and stars.